applications/HDF5Application/custom_processes/hdf5_xdmf_connectivities_writer_process.cpp

```diff
@@ -107,6 +108,32 @@ void XdmfConnectivitiesWriterProcess::CreateXdmfConnectivities(const std::string
 
     KRATOS_CATCH("");
 }
+    
+void XdmfConnectivitiesWriterProcess::CreateXdmfPoints(
+    const std::string& rKratosNodeIdsPath,
+    const std::string& rXdmfNodeIdsPath) const
+{
+    KRATOS_TRY
+
+    if (mpFile->IsDataSet(rKratosNodeIdsPath + "/NodeIds")) {
+        Vector<int> kratos_ids, xdmf_ids;
+        const int block_size = mpFile->GetDataDimensions(rKratosNodeIdsPath + "/NodeIds")[0];
+        mpFile->ReadDataSet(rKratosNodeIdsPath + "/NodeIds", kratos_ids, 0, block_size);
+
+        xdmf_ids.resize(kratos_ids.size(), false);
+
+ #pragma omp parallel for
+        for (int i = 0; i < block_size; ++i) {
+                xdmf_ids[i] = mKratosToXdmfIdMap.at(kratos_ids[i]);
```

Comment:
btw why do you use `at` and not `[]` 
Afaik `at` is slower as it also does some checks